pom.xml

```diff
@@ -224,28 +224,6 @@
       </roles>
       <timezone>Europe/Madrid</timezone>
     </developer>
-    <developer>
```

Comment:
Check this conflict (in both pom), it's not a bug.

Reply:
There is no conflict